src/services/utilServices/SitesService.js

```diff
@@ -29,12 +29,33 @@ const ISOMER_ADMIN_REPOS = [
 ]
 
 class SitesService {
-  constructor({ gitHubService, configYmlService }) {
+  constructor({
+    gitHubService,
+    configYmlService,
+    usersService,
+    isomerAdminsService,
+  }) {
     this.githubService = gitHubService
     this.configYmlService = configYmlService
+    this.usersService = usersService
+    this.isomerAdminsService = isomerAdminsService
   }
 
   async getSites(sessionData) {
+    const isEmailUser = sessionData.getIsEmailUser()
+    let emailAccessibleSites = []
```

Comment:
nit: could we rename this to `retrievedSitesByEmail`? i got confused by this at first ._.

Reply:
Sure, https://github.com/isomerpages/isomercms-backend/pull/480/commits/0fdffecba8dfc13ac707441a01c702444935db99